app/demo-opentitan-earlgrey/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+edition = "2018"
+readme = "README.md"
+name = "demo-opentitan-earlgrey"
+version = "0.1.0"
+
+[features]
+#semihosting = ["kern/klog-semihosting-riscv"]
```

Comment:
I think this should be dropped as there is no klog support in upstream sys/kern anymore.

Reply:
Done.